### DISCUSSION TITLE
Tree Style Tabs + Auto Hide CSS theme - is there are a way to center "new tab" sign?

### DISCUSSION THREAD

**Discussion by vovavili** [piroor/treestyletab#3039] [Q&A]
Hello, developers of this wonderful extension!

[This is driving my OCD crazy](https://imgur.com/a/3C26XVg). Is there a way to fix this and make it centered in line with all other tabs?

I've used [default autohide theme](https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#full-auto-showhide-theme), my entire userChrome.css can be found [here](https://pastebin.com/6GkMBf2D).

Thank you all very much in advance!

**Comment by piroor** [comment]
Knowledge about customization with user style sheet should not be tracked in the issue tracker (it should track only bugs, feature requests, and other development tasks) so I've converted this to a discussion topic.

**Reply by vovavili** [reply]
My bad.

**Reply by piroor** [reply]
No problem!

**Comment by vovavili** [comment]
I resolved this. [One wonderful individual on reddit has managed to help me with this](https://www.reddit.com/r/FirefoxCSS/comments/qtw3ji/tree_style_tabs_auto_hide_css_theme_is_there_are/). I'll add his contributions to the wiki :)
